modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java

```diff
@@ -446,9 +447,9 @@ private CompletableFuture<Integer> saveUpdate(UpdateProducer updateProducer, int
     public List<SystemView<?>> systemViews() {
         return List.of(
                 createSystemViewsView(),
-                createSystemViewColumnsView(),
-                createSystemViewZonesView(),
-                createSystemViewIndexesView()
+                createColumnsView(),
```

Comment:
It seems that `createSystemViewColumnsView` was a correct name as it was a view presenting columns of system views.